### ISSUE TITLE
JRuby 9.3+ raises "Found multiple supers in java ctor" if a plugin does  not found.

### ISSUE THREAD

**Issue by hiroyuki-sato** [embulk/embulk#1513]
Issue Type: Bug Report
-----------------------

* Write the following environmental information.
  * OS version: macOS 12.3.1
  * Java version: 1.8
  * Embulk version: 0.10.36-SNAPSHOT(39d8df31ebb1f19379881d241d38a83f37031a5c) 

JRuby 9.3+ raises "Found multiple supers in java ctor" if a plugin does not found.

```yaml
in:
  type: postgresql2 # no exist plugin.
  host: localhost
  user: user
  password: password
  database: embulk_test
  table: inc_test
out:
  type: stdout
```

JRuby 9.3.4.0

```
Error: InputPlugin 'postgresql2' is not found.
Maven repository specified is not found at "/path/to/embulk/embulk-dev-helpers/build/embulk_0.10.36-SNAPSHOT_jruby_9.3.4.0/lib/m2/repository" ("/path/to/embulk/embulk-dev-helpers/build/embulk_0.10.36-SNAPSHOT_jruby_9.3.4.0/lib/m2/repository").
org.jruby.exceptions.RuntimeError: (RuntimeError) Found multiple supers in java ctor
```

JRuby 9.2.20.1

```
Error: InputPlugin 'postgresql2' is not found.
Maven repository specified is not found at "/path/to/embulk/embulk-dev-helpers/build/embulk_0.10.36-SNAPSHOT_jruby_9.2.20.1/lib/m2/repository" ("/path/to/embulk/embulk-dev-helpers/build/embulk_0.10.36-SNAPSHOT_jruby_9.2.20.1/lib/m2/repository").
org.jruby.proxy.org.embulk.config.ConfigException$Proxy1: Unknown input plugin 'postgresql2'. embulk/input/postgresql2.rb is not installed. Run 'embulk gem search -rd embulk-input' command to find plugins.
```

* Ref1: https://github.com/jruby/jruby/wiki/CallingJavaFromJRuby#subclassing-a-java-class
* Ref2: https://gist.github.com/hiroyuki-sato/972d1b02a1b3407c09795274cf6aa449

This error may relate to the following code.

https://github.com/embulk/embulk/blob/d7be6aadd4e8f284a66e069d07a47f175a06082e/embulk-ruby/lib/embulk/error.rb#L5-L13

**Comment by hiroyuki-sato** [comment]
```java
class JRubyTest {

  private String msg;
  public JRubyTest() {
    System.out.println("no args");
    this.msg = "default";
  }

  public JRubyTest(String msg) {
    System.out.println("with args");
    this.msg = msg;
  }

  public String hello(){
    return this.msg;
  }
}
```

```ruby
java  -jar /path/to/jruby-complete-9.3.4.0.jar -S 'irb'
irb(main):001:0> $CLASSPATH << "/path/to/class"
=> ["..", "file:/path/to/class/"]
irb(main):002:0> java_import "JRubyTest"
=> [Java::Default::JRubyTest]
irb(main):003:0> Java::Default::JRubyTest.new
no args
=> #<Java::Default::JRubyTest:0x5befbac1>
irb(main):004:0> Java::Default::JRubyTest.new("test")
with args
=> #<Java::Default::JRubyTest:0x43d3aba5>
```

**Comment by kakoni** [comment]
Hi @hiroyuki-sato!

Just bumped into this when trying test/run jruby based plugins with recent JRuby (I'm on M1 Mac so it seems 9.3.3.0 is minimum that's working somehow properly). Would be it be possible to get this merged into v0.10.38?

**Comment by hiroyuki-sato** [comment]
Hello, @kakoni 
 #1514 fixes this issue. It will be in 0.10.39 or something.
(0.10.38 (next version) will fix another issue. Please wait for a while.)
